components/pagination/stories/pagination.stories.js

```diff
@@ -70,3 +89,17 @@ WithForcedColors.parameters = {
 		modes: disableDefaultModes
 	},
 };
+
+// ********* DOCS ONLY ********* //
+/**
+ * Pagination's explicit variant uses the text field component to represent the current page number,
+ * and action buttons for the previous and next navigation. It also displays text with the total
+ * number of pages.
+ */
+export const Explicit = Template.bind({});
```

Comment:
We should add the tags here, right (`!dev`)? To get this story out of the sidebar?

Reply:
I'm purposefully including this one on the sidebar. It's a very different type of pagination that uses different elements/markup. I think it makes sense in this case, especially since these two types of pagination have their own separate docs site pages.

Reply:
I didn't re-read your last response to me on this. sorry!